server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -91,7 +91,7 @@ func newRunner(ctx workflow.Context, request Request, tfWorkflow terraform.Workf
 		GithubCheckRunCache: checkRunCache,
 	}
 	revisionQueue := queue.NewQueue(func(ctx workflow.Context, d *queue.Deploy) {
-		lockStateUpdater.UpdateQueuedRevisions(ctx, d)
+		lockStateUpdater.UpdateQueuedRevisions(ctx, d, request.Repo.FullName)
```

Comment:
just passing the full name here instead of the repo object to avoid import cycle